draft-ietf-tls-esni.md

```diff
@@ -850,21 +850,21 @@ If the client offers the "ech_is_inner" extension ({{is-inner}})
 in addition to the "encrypted_client_hello" extension, the server MUST abort
 with an "illegal_parameter" alert.
 
-First, the server collects a set of candidate ECHConfigs. This set is
-determined by one of the two following methods:
+First, the server collects a candidate ECHConfigList. This list is determined
```

Comment:
I think this section should just be "ECHConfigs" or "ECHConfig values". The server doesn't actually need to serialize them or anything.

Also now that we also have a better word, maybe {{real-ech}} should now be a bit more precise and say something like:
"To offer ECH, the client first chooses a suitable ECHConfig from the published ECHConfigList."

Reply:
(See latest change)